contrib/clojure-package/examples/pre-trained-models/src/pre_trained_models/predict_image.clj

```diff
@@ -92,7 +92,7 @@
 
 (comment
 
-  (predict "https://raw.githubusercontent.com/dmlc/web-data/master/mxnet/doc/tutorials/python/predict_image/cat.jpg")
+  (predict "https://raw.githubusercontent.com/dmlc/web-data/master/mxnet/doc/tutorials/python/predict_image/cat.jpg" true)
```

Comment:
What does this `true` mean in here?

Reply:
It's for the example - it controls whether the image url displays in a popup for you view 
https://github.com/apache/incubator-mxnet/blob/6fd245d1ce28a7bc09b83aeb2a1233f085210037/contrib/clojure-package/examples/pre-trained-models/src/pre_trained_models/predict_image.clj#L42

Reply:
nit: not a big deal since it's a helper fn in an example, but it'd make sense to use an options map or kw arg (`{:display true}` or `:display true`). [outside this PR though, also]

Reply:
yes agree